frontend/src/components/Plan/Plan.tsx

```diff
@@ -0,0 +1,69 @@
+import { useParams } from 'react-router-dom';
+import { icon } from '@fortawesome/fontawesome-svg-core/import.macro'
+import styles from './Plan.module.css'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { PlanDescription } from './PlanDescription'
+import { Memberslist } from './MembersList'
+import { fetchPlanData } from './hook/fetchPlanData'
+import { loadingStatus } from '../../utils/loadingStatus'
+import { PlanIdeaList } from './PlanIdeaList'
+
+export interface Member {
+  leader: string
+  friends: FriendList[]
+}
+
+export interface FriendList {
```

Comment:
nit: this name implies that it types a list, but it is just an item in that list. this should probably be called `Friend` and then on line 13 it would be `friends: Friend[]`

Reply:
Oh right, I changed it to `Friend`